### PR TITLE
Target downgraded to iOS 10

### PR DESCRIPTION
**Pull request by ennioma**
If this PR will be fine, I'm going to create a release 4.0.2 that compiles on Xcode 10 and support swift 4.2 and minimum iOS target = iOS 10

### CODE REVIEW

**Comment by ennioma** [comment]
@robbiet480 please could you take a look at this PR?

Basically from the next release I'm going to support
Xcode 10
swift 4.2
Minimum target iOS 10

File: Example/arek_example.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -565,6 +549,7 @@
 				CODE_SIGN_STYLE = Manual;
 				DEVELOPMENT_TEAM = "";
 				INFOPLIST_FILE = arek_example/Info.plist;
+				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
```
[the remark this answers is not in the document]
Comment (ennioma) [reply Example/arek_example.xcodeproj/project.pbxproj]: Oh well that's the project example

File: Example/arek_example.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -585,6 +570,7 @@
...
```
[the remark this answers is not in the document]
Comment (ennioma) [reply Example/arek_example.xcodeproj/project.pbxproj]: As above, that's only the project demo app

**Comment by robbiet480** [comment]
LGTM!

**Pull request merged**